# Memory leak fix
#
# Copyright (C) 2006 - 2007 Nokia Corporation.
# This file is distributed under the terms of GNU LGPL license, either version 2
# of the License, or (at your option) any later version.
#
diff -ru gtkhtml-3.13.91/src/htmlengine.c gtkhtml-3.13.91.mod0/src/htmlengine.c
--- gtkhtml-3.13.91/src/htmlengine.c	2007-02-12 08:57:38.000000000 +0200
+++ gtkhtml-3.13.91.mod0/src/htmlengine.c	2007-02-15 12:41:23.000000000 +0200
@@ -1550,6 +1550,7 @@
 	g_free (data);
 	g_free (classid);
 	g_free (name);
+	html_element_free (element) ;
 }	
 
 
